### PR TITLE
add miss constructor

### PR DESCRIPTION
**Pull request by zhoudayang**

### CODE REVIEW

File: Resolver.cc [hunk 1]
```diff
@@ -52,6 +52,20 @@ Resolver::Resolver(EventLoop* loop)
   ::dns_set_opt(ctx_, DNS_OPT_TIMEOUT, 2);
 }
 
+
```
Comment (chenshuo) [inline Resolver.cc]: remove this empty line.

File: Resolver.cc [hunk 2]
```diff
@@ -52,6 +52,20 @@ Resolver::Resolver(EventLoop* loop)
...
+Resolver::Resolver(EventLoop *loop, const InetAddress &nameServer):
+    loop_(loop),
+    ctx_(NULL),
+    fd_(-1),
+    timerActive_(false)
+{
+    init_udns();
+    ctx_ = ::dns_new(NULL);
+    assert(ctx_ !=NULL);
```
Comment (chenshuo) [inline Resolver.cc]: Need a space between '!=' and NULL.

File: Resolver.cc [hunk 3]
```diff
@@ -52,6 +52,20 @@ Resolver::Resolver(EventLoop* loop)
...
+    ::dns_add_serv_s(ctx_,nameServer.getSockAddr());
```
Comment (chenshuo) [inline Resolver.cc]: Need one space after each comma (,), same for next line.

**Review by chenshuo — CHANGES_REQUESTED** [review]

**Comment by zhoudayang** [comment]
Thank you! I'm sorry for my carelessness.

**Pull request merged**